### PR TITLE
Search by term: extend to entries url

### PR DESCRIPTION
**Pull request by Kdecherf**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | idk
| Documentation | no
| Translation   | no
| Fixed tickets | -
| License       | MIT

This PR extends search to entries url to provide an alternative way to domain filtering.

### CODE REVIEW

File: tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php [hunk 1]
```diff
@@ -1093,5 +1093,17 @@ public function testSearch()
         $crawler = $client->submit($form, $data);
 
         $this->assertCount(0, $crawler->filter('div[class=entry]'));
+
+        // test domain on list of all articles
+        $crawler = $client->request('GET', '/all/list');
+
+        $form = $crawler->filter('form[name=search]')->form();
+        $data = [
+            'search_entry[term]' => 'domain', // a string not available in the database
```
Comment (j0k3r) [inline tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php]: You might change the comment and maybe use a real domain name instead?

Comment (Kdecherf) [inline tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php]: Yes, I will update the comment to say that the filtering is not limited to domains but is applied to the whole url.

For the value used for the test, I took the value from `testFilterOnDomainName`.

**Comment by Kdecherf** [comment]
The current change is expected to fail with PostgreSQL. It will help confirm if there is case sensitivity issue on the search part

**Comment by Kdecherf** [comment]
Ready for review

**Comment by j0k3r** [comment]
Great!

**Pull request merged**